editor/js/editor.js

```diff
@@ -77,7 +77,7 @@
                 var styleElements = shadow.querySelectorAll('style');
 
                 for (var styleElement in styleElements) {
-                    if (styleElements.hasOwnProperty(styleElement)) {
+                    if (styleElements.hasOwnProperty(styleElement) && styleElements[styleElement]) {
```

Comment:
I think the real error is from line 76 😕

Reply:
#542 Thanks for the head-ups I missed that one.